nomad/state/state_store_test.go

```diff
@@ -316,7 +403,7 @@ func TestStateStore_UpsertPlanResults_PreemptedAllocs(t *testing.T) {
 	preempted, err := state.AllocByID(ws, preemptedAlloc.ID)
 	require.NoError(err)
 	require.Equal(preempted.DesiredStatus, structs.AllocDesiredStatusEvict)
-	require.Equal(preempted.DesiredDescription, fmt.Sprintf("Preempted by allocation %v", alloc.ID))
+	require.Equal(preempted.DesiredDescription, fmt.Sprintf("Preempted by alloc ID %v", alloc.ID))
```

Comment:
Is there a reason we changed this string? Old version seems fine.

Reply:
The older version of string in the test was incorrect. The source code has always used "alloc ID". Earlier the test used to setup this data (with the incorrect string), whereas now it is denormalized in the source code and hence, the change.